libraries/common/components/InfiniteContainer/index.jsx

```diff
@@ -40,6 +41,7 @@ class InfiniteContainer extends Component {
     requestHash: null,
     totalItems: null,
     wrapper: 'div',
+    promiseBased: false,
```

Comment:
I spent some time thinking about this name but I can not think of anything better. Maybe "enablePromiseBasedLoading" to make it more specified.

Reply:
updated